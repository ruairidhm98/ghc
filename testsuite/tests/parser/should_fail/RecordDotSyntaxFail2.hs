{-# LANGUAGE NoRecordDotSyntax #-}

data Foo = Foo { foo :: Bar }
data Bar = Bar { bar :: Baz }
data Baz = Baz { baz :: Quux }
data Quux = Quux { quux :: Int }

no :: Foo -> Foo
no = Foo { bar.baz = Quux { quux = 42 } } } }
  -- Syntax error: RecordDotSyntax is not enabled
