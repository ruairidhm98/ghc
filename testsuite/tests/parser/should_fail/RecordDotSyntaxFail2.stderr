RecordDotSyntaxFail2.hs:9:15: parse error on input ‘.’
